core/src/test/kotlin/org/evomaster/core/search/impact/genemutationupdate/IntegerGeneMutationUpdate.kt

```diff
@@ -0,0 +1,118 @@
+package org.evomaster.core.search.impact.genemutationupdate
+
+import com.google.inject.Injector
+import com.google.inject.Key
+import com.google.inject.Module
+import com.google.inject.TypeLiteral
+import com.netflix.governator.guice.LifecycleInjector
+import org.evomaster.core.BaseModule
+import org.evomaster.core.EMConfig
+import org.evomaster.core.search.algorithms.MioAlgorithm
+import org.evomaster.core.search.matchproblem.*
+import org.evomaster.core.search.service.Archive
+import org.evomaster.core.search.service.mutator.EvaluatedMutation
+import org.evomaster.core.search.service.mutator.MutatedGeneSpecification
+import org.evomaster.core.search.service.mutator.StandardMutator
+import org.evomaster.core.search.service.mutator.genemutation.ArchiveGeneMutator
+import org.evomaster.core.search.service.mutator.genemutation.mutationupdate.LongMutationUpdate
+import org.evomaster.core.search.tracer.ArchiveMutationTrackService
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions.*
+
+/**
+ * created by manzh on 2020-09-10
+ */
+class IntegerGeneMutationUpdate {
```

Comment:
remember that all test classes MUST end with `Test`... otherwise Maven will not execute them. if this is not supposed to be executed directly, eg used as a superclass for another test class, then it should be marked as `abstract`